builder/steps/gen-dockerfile/contents/.clang-format

```diff
@@ -0,0 +1,3 @@
+Language: JavaScript
+BasedOnStyle: Google
+ColumnLimit: 100
```

Comment:
No.  It just seemed to fit the code well.  I've changed it to 80 since that it what I usually use, and because I need to do some line splitting anyway (unless you think 120 is better).